budget/utils.py

```diff
@@ -1,11 +1,12 @@
 import re
 import inspect
 
+from io import BytesIO, StringIO
```

Comment:
I guess you need to use six there too. https://pythonhosted.org/six/#six.BytesIO